### PR TITLE
Add options to enable / disable What4 hash consing

### PR DESCRIPTION
**Pull request by chameco**

### CODE REVIEW

File: src/SAWScript/Interpreter.hs [hunk 1]
```diff
@@ -520,15 +521,25 @@ enable_crucible_profiling f = do
   rw <- getTopLevelRW
   putTopLevelRW rw { rwProfilingFile = Just f }
 
+disable_crucible_profiling :: TopLevel ()
```
Comment (kquick) [inline src/SAWScript/Interpreter.hs]: This seems unrelated.

Comment (chameco) [reply src/SAWScript/Interpreter.hs]: This was a reorder, so that `enable_crucible_profiling` and `disable_crucible_profiling` would be adjacent. Not directly related to this, but I figured I'd fix it since I was working with the adjacent code.

Comment (chameco) [reply src/SAWScript/Interpreter.hs]: Perhaps not the best for commit history though. In the future I'll try to split things like that into separate PRs if it's preferable.

Comment (kquick) [reply src/SAWScript/Interpreter.hs]: No, that was fine: I misread the diff and was looking for a big removed chunk whereas git diff actually noted it as name changes below. Sorry for the confusion and this was a good patch.
